src/Gui/Tree.cpp

```diff
@@ -855,19 +860,35 @@ void TreeWidget::contextMenuEvent (QContextMenuEvent * e)
         DocumentObjectItem* objitem = static_cast<DocumentObjectItem*>
             (this->contextItem);
 
+        // check that the selection is not across several documents
+        bool acrossDocuments = false;
+        auto SelectedObjectsList = Selection().getCompleteSelection();
+        QList<QTreeWidgetItem*> items = this->selectedItems();

```

Comment:
Items is nowhere used below. In line 893 the function is used again
`auto selItems = this->selectedItems();`
where it's used then.

Reply:
thanks, fixed